docker-compose.yml

```
@@ -0,0 +1,153 @@
+  postgres_test_data:
+    driver: local
+  redis_data:
+    driver: local
+  prometheus_data:
+    driver: local
+  grafana_data:
+    driver: local
+
+networks:
+  ccui-network:
+    driver: bridge
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)